frontend/src/pages/Home/components/Overview/overview.module.scss

```diff
@@ -64,29 +64,45 @@
   }
 }
 
-@media screen and (min-width: 550px) and (max-width: 800px) {
+@media screen and (max-width: 800px) {
   .overview {
+    flex-direction: column;
     padding: 1.8rem 0;
   }
+  .overview > div {
+    padding-right: 0;
+  }
 
   .overview p {
     font-size: 1.3rem;
-  }
-
-  .overview > div {
-    padding-right: 0;
+    text-align: center;
   }
 
   .overview img {
+    order: -1;
     min-width: auto;
     width: 180px;
-    margin-left: 11px;
+  }
+
+  .overview h1 {
+    text-align: center;
```

Comment:
this shouldn't be centered aligned we want it just above the dashed line[i.e same as it was before]

Reply:
But it was   aligned to center from  begining , kindly check it at your repo